### PR TITLE
BT Stake

### PR DESCRIPTION
**Pull request by gulshanvasnani**
PR contains following :-
1. Commands for bt stake and mint.
2. Updated readme for the commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,53 @@ node src/bin/bt.js setupUtilityBrandedToken <config.json>
...
+```bash
+# Help: 
+node src/bin/bt.js --help
+
+node src/bin/bt.js acceptStake path_to_config.json stakeRequestHash
+
+```
+
+* Replace `path_to_config.json` with a file location where config is stored.
+* Replace `stakeRequestHash` with the hash received from `requestStake` process.
```
Comment (sarvesh-ost) [inline README.md]: After this, mention that use a normal facilitator to progress.

File: src/bt_deployer.js [hunk 2]
```diff
@@ -173,19 +179,54 @@ class BTDeployer {
...
+      this.chainConfig.brandedToken.address,
+    );
+
+    const liftRestrictionTxOptions = {
+      gasPrice: this.origin.txOptions.gasPrice,
+      from: this.origin.masterKey,
+    };
+    const stakeVaultAddress = await originGateway.getStakeVault();
+
+    await brandedToken.liftRestriction(
```
Comment (sarvesh-ost) [inline src/bt_deployer.js]: Maybe this should happen in setup? 
Should we create a ticket?

File: src/bt_stake_mint.js [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+      gasLimit,
+    };
+
+    const staker = new Helpers.Staker(
+      this.origin.web3,
+      this.origin.token,
+      this.chainConfig.brandedToken.address,
+      this.chainConfig.gatewayComposerAddress,
+    );
+    console.log('calling request stake');
```
Comment (sarvesh-ost) [inline src/bt_stake_mint.js]: Please remove console.log from everywhere

File: src/bt_stake_mint.js [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+      staker,
+      this.chainConfig.brandedToken.address,
+    ).getEIP712SignHash();
+
+    let signature = EthUtils.ecsign(
+      EthUtils.toBuffer(requestHashToBeSigned),
+      EthUtils.toBuffer(this.chainConfig.workerPrivateKey),
+    );
+
+    signature = signData(requestHashToBeSigned, this.chainConfig.workerPrivateKey);
```
Comment (sarvesh-ost) [inline src/bt_stake_mint.js]: This is redundant to L163, use only one thing. Also clean npm dependency.

Comment (sarvesh-ost) [reply src/bt_stake_mint.js]: I think we should remove 163line signature.

File: src/config/chain_config.js [hunk 5]
```diff
@@ -7,21 +7,24 @@ class ChainConfig {
...
-    this.originWeb3Provider = process.env.ORIGIN_WEB3_PROVIDER || 'http://localhost:8545';
-    this.auxiliaryWeb3Provider = process.env.AUXILIARY_WEB3_PROVIDER || 'http://localhost:8547';
+    this.originWeb3Provider = process.env.ORIGIN_WEB3_PROVIDER || 'ws://localhost:8547';
+    this.auxiliaryWeb3Provider = process.env.AUXILIARY_WEB3_PROVIDER || 'ws://localhost:8548';
     this.originChainId = process.env.ORIGIN_CHAIN_ID || 3;
     this.auxiliaryChainId = process.env.ORIGIN_CHAIN_ID || 200;
     this.simpleTokenAddress = process.env.SIMPLE_TOKEN_ADDRESS || '0xca954C91BE676cBC4D5Ab5F624b37402E5f0d957';
-    this.originMasterKey = process.env.ORIGIN_MASTER_KEY || '0x6d4e5f5ca54b88fdb68a5d0e6ea5aa3869f88116';
-    this.auxiliaryMasterKey = process.env.AUXILIARY_MASTER_KEY || '0x490cc731b076e58509e1f7cee6c68b39cfe5ad62';
+    this.originMasterKey = process.env.ORIGIN_MASTER_KEY || '0x6c92d481eb56badef931482ef6bf6c8fbf2606e3'; // '0x0efBe7A92D98DA8203cD3FBF4E72B8a1dc4577E7';
```
Comment (sarvesh-ost) [inline src/config/chain_config.js]: You shouldn't change default values, Please revert.

File: src/bt_deployer.js (hunk 2 above)
Comment (gulshanvasnani) [reply src/bt_deployer.js]: Yes, we should have it in setup.
Created ticket :-
`https://github.com/OpenST/brandedtoken.js/issues/125`

Comment (sarvesh-ost) [reply src/bt_deployer.js]: Thank you 👍

**Pull request merged**